index.html

```diff
@@ -377,7 +377,7 @@
                         </dd>
                     <dt><dfn>getCoalescedEvents</dfn></dt>
                         <dd>
-                            <p>Returns a sequence of all <code>PointerEvents</code> that were coalesced into the event. The coalesced events will have the same value as the event in attributes like <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, and particularly <code>target</code>. But other attributes might be different. The event itself will have the greatest <a href="https://dom.spec.whatwg.org/#dom-event-timestamp">timeStamp</a> ([[!WHATWG-DOM]]) among the events in the sequence and the first event in the sequence will have the smallest timeStamp. None of the events in the returned sequence will have coalesced events. When an event has no coalesed event this function returns an empty sequence.</p>
+                            <p>Returns a sequence of all <code>PointerEvents</code> that were coalesced into the (original) event. The following attributes of the coalesced events will always have the same value as the original event: <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, and particularly <code>target</code>. But other attributes might be different. The events in the sequence will have increasing <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a> ([[!WHATWG-DOM]]). So the first event will have the smallest <code>timeStamp</code>. Also the original event's <code>timeStamp</code> will be greater than or equal to the last event's <code>timeStamp</code> in the sequence. None of the events in the sequence will have coalesced events. When an event has no coalesed event this function returns an empty sequence.</p>
```

Comment:
Another thought: do we need a note that two events for different pointerIds could contain interleaved historical events? E.g. the input sequence:
  [p1-move1, p2-move1, p2-move2, p1-move2]
could be received as:
  [p2-move2 (containing coalesced p2-move1), p1-move2 (containing p1-move1)].
This may not be obvious from the current text.